src/Util/SetupCommandTrait.php

```diff
@@ -137,7 +137,14 @@ protected function bootSetupSubsystem(InputInterface $input, OutputInterface $ou
       $setup->getModel()->cmsBaseUrl,
     ]);
     if ($input->getOption('db')) {
-      $setup->getModel()->db = DbUtil::parseDsn($input->getOption('db'));
+      $model = $setup->getModel();
+      // Set db and also set cmsDb to the same db if standalone
+      $model->db = DbUtil::parseDsn($input->getOption('db'));
+
+      // todo: artfulrobot: we should only do this for standalone
+      if ('standalone fixme' && empty($model->cmsDb)) {
+        $model->cmsDb = $setup->getModel()->db;
+      }
```

Comment:
Is the CMS database used somewhere by cv? If not you could skip this step.

Reply:
@artfulrobot I was hoping we could get these things merged to prevent MC's. It seemed the main issue was this `standalone fixme` conditional. After poking around, I think you're probably right that it's easier to oblige it in this way. I've pushed an update to use a real conditional.

As y'all indicate, there is a bit of impedance mismatch since the existence of a prior CMS was an original assumption of the `Civi\Setup` library -- and it's not the same for "Standalone". We may want to talk/shift more to account for that, but (for the moment) I think it's more useful to consolidate a bit so that we don't have as many patches floating around.